frontend/src/components/MethodologyDialog/MethodologyDialog.tsx

```diff
@@ -11,6 +12,27 @@ type Props = {
 export let MethodologyDialog: React.StatelessComponent<Props> = ({
   isOpen, onCloseClick
 }) => {
+  let emailLink = SecureLink('mailto:encompass@bayesimpact.org?subject=About%20Encompass', 'encompass@bayesimpact.org')
```

Comment:
Alternatively, we could start creating a link.ts file to put reusable links together